src/Apps/CollectorProfile/Routes/Saves2/Components/SavesHeader.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react"
+import { Flex, Text, Spacer, Button, Join } from "@artsy/palette"
+import { CreateNewListModalContainer } from "./CreateNewListModal"
+import { Media } from "Utils/Responsive"
+import { CreateCollectionMutationResult } from "Apps/CollectorProfile/Routes/Saves2/types"
+import { useToasts } from "@artsy/palette"
+import { useTranslation } from "react-i18next"
+
+export const SavesHeader: React.FC = () => {
+  const { t } = useTranslation()
+  const { sendToast } = useToasts()
+  const [modalIsOpened, setModalIsOpened] = useState(false)
+
+  const handleCreateNewListClick = () => {
+    setModalIsOpened(true)
+  }
+
+  const handleComplete = (_result: CreateCollectionMutationResult) => {
+    setModalIsOpened(false)
+
+    sendToast({
```

Comment:
`sendToast` is only a temporary solution. In the next tickets we will add a modal to select artworks